packages/rspack/tests/configCases/builtin-swc-loader/react-refresh-false/webpack.config.js

```diff
@@ -5,11 +5,12 @@ module.exports = {
 				test: /\.js$/,
 				loader: "builtin:swc-loader",
 				options: {
-					rspackExperiments: {
-						react: {
-							refresh: false
-						}
-					}
+					// TODO: add back this until `@rspack/plugin-react-refresh` is finished
```

Comment:
maybe should remove this comment, since another separate loader seems a better option

Reply:
Yes, I mean we can switch to `@rspack/plugin-react-refresh`